src/__tests__/replset-test.ts

```diff
@@ -50,7 +51,7 @@ describe('multi-member replica set', () => {
   afterEach(async () => {
     if (!replSet) return;
     await replSet.stop();
-    replSet = (null: any);
+    // replSet = (null: any); // TODO : why do we need this ?
```

Comment:
@nodkz

Reply:
Maybe @jloveridge can explain.
But I think that we can remove this line.